rfcs/rfc-005-branches-tags-and-channels.md

```diff
@@ -0,0 +1,52 @@
+```
+========================= master branch (edge channel) =======================>
+         \                      \                     \
+          \___v0.7.0 tag         \                     \
+           \                      \         v0.9.0 tag__\
+            \          v0.8.0 tag__\                     \
+ v0.7.1 tag__\                      \                 v0.9 branch (beta channel)
+              \___v0.7.2 tag         \___v0.8.1 tag
+               \                      \
+                \                      \
+           v0.7 branch         v0.8 branch (stable channel)
+
+```
+
+## Branches and Tags
+
+### master branch
+All new development occurs on the `master` branch.
+
+Bug fixes that affect a `vX.Y` branch are first made on `master`.  Once the bug
+fix lands on `master` it is cherry-picked into the `vX.Y` branch and potentially
```

Comment:
Why potentially `vX.Y+1`? Why not `vX.Y+1` first, then `vX.Y` and under the same justification for landing in `master` before a release branch?

Reply:
Yeah, sure.  Ideally we propagate from from least stable to most stable so that makes more sense.